aana/deployments/hf_blip2_deployment.py

```diff
@@ -20,11 +20,11 @@ class HFBlip2Config(BaseModel):
     """The configuration for the BLIP2 deployment with HuggingFace models.
 
     Attributes:
-        model (str): the model ID on HuggingFace
-        dtype (str): the data type (optional, default: "auto"), one of "auto", "float32", "float16"
-        batch_size (int): the batch size (optional, default: 1)
-        num_processing_threads (int): the number of processing threads (optional, default: 1)
-        max_new_tokens (int): The maximum numbers of tokens to generate. (optional, default: 64)
+        model (str): The model ID on HuggingFace.
```

Comment:
In some models we have `model_id` and in others we have `model`. Can we change all of them to be the same? It can cause problem for existed projects.

Reply:
It's a good idea to make it more consistent. I changed it to model_id. You are right, it will cause problems with existing projects that's why I added an alias so it's possible to use the old "model" name for the deployments that used that but we should switch to model_id from now.